09-11_git_ex_script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#Script to set up a directory for CS 640 git exercise.
+
+#Christopher M. Shinn cmshinn@dons.usfca.edu
+
+#2017 September 13
+
+#Set up directories
+
+echo "Setting up directories!"
+
+mkdir 2017-09-13_git_flow_exercise_CS640
+
+cd 2017-09-13_git_flow_exercise_CS640
```

Comment:
No need to make a directory like this for this project - if you leave it out, then the rest of the code could be reused in any project with a similar structure.